### ISSUE TITLE
Missing Ben's Bio and headshot on the contact us page & bios page

### ISSUE THREAD

**Issue by kmonterr** [18F/nsf-sbir#1647]
Hi @seakhtar -

I hope you had a nice holiday and a great start to 2020!

We noticed that Ben is no longer to be found on the Contact Us page or the Bios page. While he doesn't have a topic at the moment, he should still be on the contact us page and the bios page.  Thank you so much for adding him back!
Kelly

**Comment by seakhtar** [comment]
Hi @kmonterr -

Thanks for the heads up, I'll get him posted again!

Hope you enjoyed your holiday and got some relaxation time in :)

**Comment by seakhtar** [comment]
Hi @kmonterr ,

Sorry for the delay on this! Running into a familiar issue. Not sure if you remember me talking about this, but the contact/bio pages are created with an algorithm using the information on the Portfolio page. When I added Ruth Shuman (who also isn't on the Portfolio page), I managed to get around that constraint by manually tacking her on to the end of the list. The thing is, we lucked out that Ruth was already at the end of the alphabet. With Ben, he's supposed to come before Rick, but I can't break the algorithm to stick him in there. 

The worst part is, their names are so similar that even adding Ben to the end of the list sort of makes it look like we just messed up our alphabetical sorting. I'm trying to find a way around all this, but I wanted to just give you an update as to why it's taking so long. I'll keep you posted!

**Comment by kmonterr** [comment]
Thanks for the heads up!! :)

**Comment by seakhtar** [comment]
Hi @kmonterr ,

So I've been trying to find a way around this, and nothing I've tried is working. If you'd really like to keep the alphabetical order, I can change the page so that it is no longer dynamically generated and just manually list out all of the PDs. We'd have more control over how they display, but we'd have to go in to the Contact and Bio pages each time we want to make a change to the PDs. 

Or if listing him alphabetically is not as important, we can just add him to the end of the list like we did for Ruth. 

Just let me know which you'd prefer and I'll get this finished up!
